### PR TITLE
Add Acconeer XM122 motion sensor support

### PR DESCRIPTION
**Pull request by jbjornstedt**

### CODE REVIEW

File: custom_components/ble_monitor/ble_parser/__init__.py [hunk 1]
```diff
@@ -303,6 +304,10 @@ def parse_data(self, data):
                         # AltBeacon
                         sensor_data, tracker_data = parse_altbeacon(self, man_spec_data, comp_id, mac, rssi)
                         break
+
+                    elif man_spec_data[0] == 0x12 and comp_id == 0xacc0:  # Acconeer
```
Comment (Ernst79) [inline custom_components/ble_monitor/ble_parser/__init__.py:308]:
```suggestion
                    elif man_spec_data[0] == 0x12 and comp_id == 0xACC0:  # Acconeer
```

File: custom_components/ble_monitor/const.py [hunk 2]
```diff
@@ -1120,7 +1121,6 @@ class BLEMonitorBinarySensorEntityDescription(
     'iBeacon'                 : 'Apple',
     'AltBeacon'               : 'Radius Networks',
     'EClerk Eco'              : 'Relsib',
-    'Air Mentor Pro 2'        : 'Air Mentor',
```
Comment (Ernst79) [inline custom_components/ble_monitor/const.py:1123]: Why is Air Mentor Pro 2 delete? I think you should add Acconeer.

**Comment by Ernst79** [comment]
Thanks, I have added a few small changes.

File: custom_components/ble_monitor/ble_parser/__init__.py (hunk 1 above)
Comment (jbjornstedt) [reply custom_components/ble_monitor/ble_parser/__init__.py]: Done!

File: custom_components/ble_monitor/const.py (hunk 2 above)
Comment (jbjornstedt) [reply custom_components/ble_monitor/const.py]: Sorry about that. Rebase conflict fix gone bad

File: custom_components/ble_monitor/const.py [hunk 3]
```diff
@@ -1182,7 +1182,6 @@ class BLEMonitorBinarySensorEntityDescription(
 # Selection list for report_uknown
 REPORT_UNKNOWN_LIST = [
     "Off",
-    "Air Mentor"
```
[the remark this answers is not in the document]
Comment (jbjornstedt) [reply custom_components/ble_monitor/const.py:1185]: Sorry about that. Rebase conflict fix gone bad

**Review by Ernst79 — APPROVED** [review]

**Pull request merged**